plugins/modules/sonic_prefix_lists.py

```diff
@@ -227,6 +231,95 @@
 # sonic#
 # (no IPv6 prefix-list configuration present)
 #
+# ***************************************************************
+# Using "overriden" state to override configuration
+#
+# Before state:
+# ------------
+#
+# sonic# show running-configuration ip prefix-list
+# !
+# ip prefix-list pfx1 seq 10 permit 1.2.3.4/24 ge 26 le 30
+# ip prefix-list pfx1 seq 20 deny 1.2.3.12/26
+# ip prefix-list pfx1 seq 30 permit 7.8.9.0/24
+#
```

Comment:
Please modify this example to illustrate the fact that using "overidden" state will cause all prefix lists that aren't specified in the playbook to be deleted.

Adding one or two additional prefix lists (that get deleted) would show this clearly.

Reply:
Done

Reply:
Thank you.

Resolved.